### ISSUE TITLE
SD 2.0 ERROR ?

### ISSUE THREAD

**Issue by redsparrowgit** [TheLastBen/fast-stable-diffusion#671]
![image](https://user-images.githubusercontent.com/119093900/204140268-3f57a831-6c58-424a-9935-d2c7d385f43b.png)

**Comment by TheLastBen** [comment]
in colab : Runtime-> Change runtime type -> Runtime shape -> High-RAM

**Comment by redsparrowgit** [comment]
Yes I know, however it seems to be available only in the paid version , meanwhile $10-30 in my country is a monthly salary :)
![image](https://user-images.githubusercontent.com/119093900/204141200-18531242-2a9d-497c-9109-e593e302b85e.png)

**Comment by redsparrowgit** [comment]
This repository works without google drive . https://colab.research.google.com/github/camenduru/stable-diffusion-webui-colab/blob/main/stable-diffusion-v2-webui-colab.ipynb

![image](https://user-images.githubusercontent.com/119093900/204141551-64de8307-10b1-464b-83a0-4292e5148cb8.png)

Maybe it would be possible to flip something to this temporary disk ?

**Comment by TheLastBen** [comment]
colab won't let you use virtual memory, I tried.

**Comment by TheLastBen** [comment]
I'll check it out

**Comment by redsparrowgit** [comment]
Thank You :)

**Comment by TheLastBen** [comment]
That was just a lucky run, tried it, same error, can you reproduce that everytime ?

**Comment by Kaz03** [comment]
![image](https://user-images.githubusercontent.com/11291649/204143821-83786008-afbc-46de-a270-3503d657a64e.png)

2/2 runs so far on colab mentioned here. Will try a few more times

**Comment by redsparrowgit** [comment]
"That was just a lucky run, tried it, same error, can you reproduce that everytime ? "

Yes every time working on free colab,

**Comment by TheLastBen** [comment]
that's weird, I'll dig more.

**Comment by patriciouca** [comment]
I have the same issue.

**Comment by elBlacksmith** [comment]
> I checked this: https://colab.research.google.com/github/camenduru/stable-diffusion-webui-colab/blob/main/stable-diffusion-v2-webui-colab.ipynb it out and eevidently it eats less ram. I don't know what is the reason for this . But working with LESS RAM
> 
> ![image](https://user-images.githubusercontent.com/119093900/204142236-b4190d6e-be87-4720-86c4-6fe29fa993b2.png)

That colab is workinf for me.
In a Pro account without extra ram or GPU, and in a normal free account.

Thanks

 @TheLastBen Here you have the RAM usage of that Colab. Maybe it helps you:

![image](https://user-images.githubusercontent.com/8599604/204148780-bc7779cb-642a-4e4a-8716-48f3b5fc5952.png)

**Comment by TheLastBen** [comment]
The source of the problem is loading the files from gdrive, I will add a checkbox at the very beginning that will skip the gdrive mounting and installs everything in the colab storage.

**Comment by TheLastBen** [comment]
https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111-V2.ipynb

**Comment by redsparrowgit** [comment]
Thank you for what you have done, it is certainly better than using another colab. There is only one small problem with saving images. I noticed that this occurred much earlier in previous versions but because of gdrive it was unnecessary.

![image](https://user-images.githubusercontent.com/119093900/204162594-74e764c8-1def-4e59-ae52-4d5871797573.png)
![image](https://user-images.githubusercontent.com/119093900/204162600-9ecaed22-e75d-4d2b-875c-3fca49a0cfc6.png)

**Comment by redsparrowgit** [comment]
Thank you for what you have done, it is certainly better than using another colab. There is only one small problem with saving images. I noticed that this occurred much earlier in previous versions but because of gdrive it was unnecessary.
When click this: 
![image](https://user-images.githubusercontent.com/119093900/204162713-b5c437e0-603e-4aee-a96f-9a7fc53d6644.png)

There is a this issue.
Warning: unknown mime-type for "outputs/txt2img-images" -- using "application/octet-stream"
Error: no "view" mailcap rules found for type "application/octet-stream"
/usr/bin/xdg-open: 851: /usr/bin/xdg-open: www-browser: not found
/usr/bin/xdg-open: 851: /usr/bin/xdg-open: links2: not found
/usr/bin/xdg-open: 851: /usr/bin/xdg-open: elinks: not found
/usr/bin/xdg-open: 851: /usr/bin/xdg-open: links: not found
/usr/bin/xdg-open: 851: /usr/bin/xdg-open: lynx: not found
/usr/bin/xdg-open: 851: /usr/bin/xdg-open: w3m: not found
xdg-open: no method available for opening 'outputs/txt2img-images'

**Comment by TheLastBen** [comment]
local folder features and colab don't really work well using gradio
